virtio-devices/src/vhost_user/fs.rs

```diff
@@ -452,35 +438,63 @@ impl VirtioDevice for Fs {
             None
         };
 
-        let mut handler = VhostUserEpollHandler::new(VhostUserEpollConfig {
+        // The backend acknowledged features must contain the protocol feature
+        // bit in case it was initially set but lost through the features
+        // negotiation with the guest.
+        let backend_acked_features = self.common.acked_features
+            | (self.common.avail_features & VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits());
+
+        setup_vhost_user(
+            &mut self.vu.lock().unwrap(),
+            &mem.memory(),
+            queues.clone(),
+            queue_evts.iter().map(|q| q.try_clone().unwrap()).collect(),
+            &interrupt_cb,
+            backend_acked_features,
+            &slave_req_handler,
+        )
+        .map_err(ActivateError::VhostUserFsSetup)?;
+
+        // Run a dedicated thread for handling potential reconnections with
+        // the backend.
```

Comment:
The comments need some change, as the thread is also for slave request handling.

Reply:
Agreed!